tests/components/facebox/test_image_processing.py

```diff
@@ -201,15 +216,13 @@ def mock_face_event(event):
 
     assert len(face_events) == 1
     assert face_events[0].data[ATTR_NAME] == PARSED_FACES[0][ATTR_NAME]
-    assert (
-        face_events[0].data[fb.ATTR_CONFIDENCE] == PARSED_FACES[0][fb.ATTR_CONFIDENCE]
-    )
+    assert (face_events[0].data[fb.ATTR_CONFIDENCE] == PARSED_FACES[0][
+        fb.ATTR_CONFIDENCE])
     assert face_events[0].data[ATTR_ENTITY_ID] == VALID_ENTITY_ID
-    assert face_events[0].data[fb.ATTR_IMAGE_ID] == PARSED_FACES[0][fb.ATTR_IMAGE_ID]
-    assert (
-        face_events[0].data[fb.ATTR_BOUNDING_BOX]
-        == PARSED_FACES[0][fb.ATTR_BOUNDING_BOX]
-    )
+    assert face_events[0].data[fb.ATTR_IMAGE_ID] == PARSED_FACES[0][
+        fb.ATTR_IMAGE_ID]
+    assert (face_events[0].data[fb.ATTR_BOUNDING_BOX] == PARSED_FACES[0][
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4575399)